src/app/selectors.js

```diff
@@ -31,45 +31,6 @@ export const getModelData = (state) => {
 */
 export const getControllerData = (state) => state?.juju?.controllers;
 
-/**
```

Comment:
These selectors have been extracted to the general slice file and updated to use `createSelector`.